package.json

```diff
@@ -25,5 +25,15 @@
   "bugs": {
     "url": "https://github.com/nelsonic/hapi-login/issues"
   },
-  "homepage": "https://github.com/nelsonic/hapi-login"
+  "homepage": "https://github.com/nelsonic/hapi-login",
+  "devDependencies": {
+    "bell": "^3.0.0",
+    "hapi": "^8.5.3"
+  },
+  "dependencies": {
+    "bell": "^5.1.0",
+    "hapi": "^10.0.0",
```

Comment:
IIRC, Hapi 9.2.0 >= doesn't have much changing in API layers. I can double check.